src/Engine/Manager/Managers.cpp

```diff
@@ -39,4 +46,92 @@ void Hub::ShutDown() {
     delete particleManager;
     delete physicsManager;
     delete resourceManager;
+
+    shutdown = true;
+
+}
+
+void Hub::AddComponent(World* world, Component::SuperComponent* component, const std::type_info* componentType) {
+
+    std::string name = componentType->name();
+
+    if (name == "class Component::Animation *")
```

Comment:
These strings are compiler (and compiler version) specific. This will only work on MSVC15 specifically.